script/upgrades/MU03/MU03Checks.sol

```diff
@@ -0,0 +1,687 @@
+          poolConfigs[i].medianDeltaBreakerThreshold.unwrap(),
+          poolConfigs[i].referenceRateFeedID,
+          false
+        );
+
+        // verify smoothing factor
+        if (smoothingFactor != poolConfigs[i].smoothingFactor) {
+          console2.log(
+            "MedianDeltaBreaker smoothing factor not set correctly for rate feed %s",
+            poolConfigs[i].referenceRateFeedID
+          );
+          revert("MedianDeltaBreaker smoothing factor not set correctly for all rate feeds");
+        }
+      }
+    }
+    console2.log("\tMedianDeltaBreaker cooldown, rate change threshold and smoothing factor set correctly 🔒");
+  }
+
+  function verifyValueDeltaBreaker() public view {
+    // verify that cooldown period, rate change threshold and reference value were set correctly
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isValueDeltaBreakerEnabled) {
+        uint256 cooldown = ValueDeltaBreaker(valueDeltaBreaker).rateFeedCooldownTime(
+          poolConfigs[i].referenceRateFeedID
+        );
+        uint256 rateChangeThreshold = ValueDeltaBreaker(valueDeltaBreaker).rateChangeThreshold(
+          poolConfigs[i].referenceRateFeedID
+        );
+        uint256 referenceValue = ValueDeltaBreaker(valueDeltaBreaker).referenceValues(
+          poolConfigs[i].referenceRateFeedID
+        );
+
+        // verify coodown
+        verifyCooldownTime(
+          cooldown,
+          poolConfigs[i].valueDeltaBreakerCooldown,
+          poolConfigs[i].referenceRateFeedID,
+          true
+        );
+
+        // verify rate change threshold
+        verifyRateChangeTheshold(
+          rateChangeThreshold,
+          poolConfigs[i].valueDeltaBreakerThreshold.unwrap(),
+          poolConfigs[i].referenceRateFeedID,
+          true
+        );
+
+        // verify reference value
+        if (referenceValue != poolConfigs[i].valueDeltaBreakerReferenceValue) {
+          console2.log(
+            "ValueDeltaBreaker reference value not set correctly for rate feed %s",
+            poolConfigs[i].referenceRateFeedID
+          );
+          revert("ValueDeltaBreaker reference value not set correctly for all rate feeds");
+        }
+      }
+    }
+    console2.log("\tValueDeltaBreaker cooldown, rate change threshold and reference value set correctly 🔒");
+  }
+
+  /* ================================================================ */
+  /* ============================= Swaps ============================ */
+  /* ================================================================ */
+
+  function doSwaps() public {
+    console2.log("\n== Doing some test swaps... ==");
+    swapCeloTocUSD();
+    swapcUSDtoCelo();
+    swapCeloTocEUR();
+    swapcEURtoCELO();
+    swapCeloTocBRL();
+    swapcBrlToCELO();
+    swapBridgedUSDCTocUSD();
+    swapcUSDtoBridgedUSDC();
+    swapBridgedUSDCTocEUR();
+    swapcEURtoBridgedUSDC();
+    swapBridgedUSDCtocBRL();
+    swapcBRLtoBridgedUSDC();
+  }
```

Comment:
For all of the swaps, let's also verify that the amounts returned are within what we would consider "normal" ranges, not just that they execute.